### ISSUE TITLE
Unexpected behavior with custom regex + pronouns from Ark

### ISSUE THREAD

**Issue by AbeHandler** [slanglab/phrasemachine#21]
This is a weird corner case, but worth noting and perhaps fixing. If you are using the library with the ARK tagger you might get [pronouns tagged with](http://www.cs.cmu.edu/~ark/TweetNLP/gimpel+etal.acl11.pdf) "O".

Because phrasemachine marks tokens that are not in the coarsemap with "O" (i.e. other) this does weird things when you have a custom regex that involves pronoun tags. 

```
tokens = "I drive a red car".split()
postags = "P V D A N".split()

pronouns = "O"
phrasemachine.get_phrases(tokens=tokens, postags=postags, regex=pronouns, minlen=1)["counts"]

Counter({'drive': 1})
```

I think an easy fix is to change "O" to another, rare-r character internally. Another option (better?) is just not to fix this.

**Comment by brendano** [comment]
Yes, that's always been the case. I thought I wrote a mapping system for phrasemachine that mapped both the ARK and PTB tagsets to a standardized coarse tagset, which solves this problem. If it's not in the python version, check the R version and our earlier research versions? It's somewhere...

**Comment by brendano** [comment]
Wait, pronouns aren't intended to be included for phrasemachine. They're not used from PTB either. For many applications they don't make sense to get anyway.
